ui/src/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I haven't explored deeply -- this is just the default project structure that `ng new` creates (and it works).